lib/fs.js

```diff
@@ -590,6 +603,36 @@ function writeSync(fd, buffer, offset, length, position) {
   return result;
 }
 
+// usage:
+// fs.writev(fd, buffers, [position], callback);
+function writev(fd, buffers, position, callback) {
```

Comment:
Is there a specific reason why this doesn't have its synchronous counter part in this PR?

Reply:
I just added it